README.rst

```diff
@@ -2,9 +2,7 @@ Twisted
 =======
 
 |pypi|_
-|coverage|_
```

Comment:
Not that I disagree, but why did you remove these, out of curiosity?

Reply:
This is the codecov badge. We switched to Coveralls, no?

Reply:
We did, but they have a badge too :)

Reply:
Oh, and I guess we're going to turn off Appveyor now that we have Azure?